deliver/lib/deliver/runner.rb

```diff
@@ -270,25 +270,34 @@ def submit_for_review
     # If itc_provider was explicitly specified, use it.
     # If there are multiple teams, infer the provider from the selected team name.
     # If there are fewer than two teams, don't infer the provider.
-    def transporter_for_selected_team
+    def transporter_for_selected_team(upload: false)
       # Use JWT auth
       api_token = Spaceship::ConnectAPI.token
+      api_key = if options[:api_key].nil? && !options[:api_key_path].nil?
+                  # Load api key info if user set api_key_path, not api_key
+                  { key_id: api_token.key_id, issuer_id: api_token.issuer_id, key: api_token.key_raw, is_key_content_base64: api_token.is_key_content_base64 }
+                elsif !options[:api_key].nil?
+                  options[:api_key]
+                else
+                  nil
```

Comment:
This is the redundant `else` that the linter is warning your about in [this CI run](https://app.circleci.com/pipelines/github/fastlane/fastlane/4827/workflows/f3ddde11-9de4-4d64-990f-5d70a2184212/jobs/85161?invite=true#step-107-102)

Reply:
Oh, Thank you. I success to install 3.0.4 in my machine and applied rubocop. And I also add some original changes in https://github.com/fastlane/fastlane/pull/20631/commits/38f23ee3775e0fdf62c5aee013c6bc9842223ba9